api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/FileUploadController.java

```diff
@@ -86,19 +86,22 @@ public void init() throws ServletException {
 
 	@Override
 	protected AuthorizationRequest requiredActions(VitroRequest vreq) {
-		RequestedAction ra;
+		AccessObject ra;
+		AccessOperation ao;
```

Comment:
"ra" for AccessObject does not seem intuitive to me, I would rename these two to accessObject and accessOperation respectively.

Reply:
Renamed